src/components/context/data/DataState.ts

```diff
@@ -0,0 +1,48 @@
+import { Instrument, MeasurementVariable } from 'cheminfo-types';
+
+export interface DataState {
+  measurements: Measurements;
+}
+
+interface Measurements {
+  ir: {
+    entries: MeasurementBase[];
+  };
+  raman: {
+    entries: MeasurementBase[];
+  };
+  uv: {
+    entries: MeasurementBase[];
+  };
+  nmr1h: {
+    entries: MeasurementBase[];
+  };
+  mass: {
+    entries: MeasurementBase[];
+  };
+  other: {
+    entries: MeasurementBase[];
+  };
+}
+
+interface MeasurementBase {
+  id: string;
+  title?: string;
+  instrument?: Instrument;
+  neta: Record<string, string>;
```

Comment:
I think this should be `beta`

Reply:
Actually it should be meta but indeed it was wrong